oneoffs/distillation.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Used to create distillation training examples.
+
+Distillation is training on the value from a teacher network (z=0.7)
+instead of the supervised labels (z=1). In theory this can help because
+the values are expressable by some network.
```

Comment:
nit: this is distillation, but it doesn't actually explain this file.  propose adding another sentence like 
"this script replaces the game results in a file of TFExamples (-1 or 1) with a network's evaluation of that position" (and probably add a network to your example usage string)